src/config.ts

```diff
@@ -1,19 +1,22 @@
 export interface AppConfig {
   readonly httpUrl: string;
   readonly faucetUrl?: string;
+  readonly addressPrefix: string;
   // codeId is the wasm codeId for the name service contract on the given chain
   readonly codeId: number;
 }
 
 const local: AppConfig = {
   httpUrl: "http://localhost:1317",
   faucetUrl: "http://localhost:8000/credit",
+  addressPrefix: "cosmos",
   codeId: 2,
 };
 
 const demonet: AppConfig = {
   httpUrl: "https://lcd.demo-09.cosmwasm.com",
   faucetUrl: "https://faucet.demo-09.cosmwasm.com/credit",
+  addressPrefix: "cosmos",
```

Comment:
Shall we make a demo-10 net internally, to validate all our code before coralnet?

We don't have to publish it, but it can let us do integration tests easier.

Reply:
> Shall we make a demo-10 net internally, to validate all our code before coralnet?

Yeah, why not. Makes migration steps smaller, which is good.